library/common/glfw/embedder.cc

```diff
@@ -304,10 +365,20 @@ GLFWwindow *CreateFlutterWindow(size_t initial_width, size_t initial_height,
 
   AddPlugin(window, std::move(input_plugin));
 
+  UpdateMonitorStats(window);
+
   int width, height;
   glfwGetWindowSize(window, &width, &height);
-  GLFWwindowSizeCallback(window, width, height);
-  glfwSetWindowSizeCallback(window, GLFWwindowSizeCallback);
+  glfwSetWindowSizeCallback(window, GLFWWindowSizeCallback);
```

Comment:
FWIW, the lower-case W here was intentional, since the typename of the thing being resized is `GLFWwindow` rather than `GLFWWindow`; since it's being removed it doesn't really matter though. The mouse/cursor capitalization looks like a mistake though.

Reply:
I changed the capitalization because it was inconsistent across GLFW* functions. `GLFWKeyCallback` is already with capital K. Which should it be?

Reply:
I would argue that it should have been capital W though, because that's just the name of the callback in GLFW.
If we wanted to include the typename of the thing that's on which the callback is applied consistently it should imo be `GLFWwindowWindowSizeCallback`, `GLFWwindowFramebufferSizeCallback`, `GLFWwindowMouseButtonCallback` etc.

Reply:
We could actually still do that right now? Prefixing all the callback functions with the type on which they are called (`GLFWwindow`) instead of just `GLFW`.